tslearn/docs/examples/plot_shapelet_locations.py

```diff
@@ -14,43 +14,58 @@
 # License: BSD 3 clause
 
 import numpy
```

Comment:
It would probably be a good idea to have:

* a more explicit title (to distinguish between both gallery examples related to shapelets)
* a word about what shapelet locations are

no?

Reply:
You are right. I haven't actually made any updates in the headers of these three examples, but I will update them!

Reply:
Is it more clear like this?